src/FlightDisplay/PreFlightBatteryCheck.qml

```diff
@@ -25,6 +25,7 @@ PreFlightCheckButton {
 
     property int    failurePercent:                 40
     property bool   allowFailurePercentOverride:    false
-    property var    _batPercentRemaining:           activeVehicle ? activeVehicle.battery.percentRemaining.value : 0
+    property var    batteryValue:                   activeVehicle ? activeVehicle.battery.percentRemaining.value : 0
+    property var    _batPercentRemaining:           batteryValue ? batteryValue : 0
```

Comment:
Making this an actual NaN check would be clearer

Reply:
How do I do that? Using `_batteryValue.NaN` or `_batteryValue === NaN` doesn't seem to work.

Reply:
isNaN(_batteryValue) Qml is jscript.